src/main/java/com/bringframework/factory/RawBeanProcessor.java

```diff
@@ -0,0 +1,33 @@
+package com.bringframework.factory;
+
+import static com.bringframework.util.BeanUtils.createInstance;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import java.util.Map;
+
+/**
+ * Creates instances of beans by provided {@link BeanDefinition}s from
+ * {@link BeanDefinitionRegistry}.
+ * RawBeanProcessor is not responsible for injecting other beans,
+ * but {@link AutowiredBeanPostProcessor}.
+ */
+public class RawBeanProcessor {
+
+  private final Map<String, Object> rawBeanMap;
+
+  private final BeanDefinitionRegistry beanDefinitionRegistry;
+
+  public RawBeanProcessor(BeanDefinitionRegistry beanDefinitionRegistry,
+                          Map<String, Object> rawBeanMap) {
+    this.beanDefinitionRegistry = beanDefinitionRegistry;
+    this.rawBeanMap = rawBeanMap;
+  }
+
+  public void process() {
```

Comment:
It would be good if we have some logging during bean creation process

Reply:
Good point!
Added